apollo-compiler/src/main/kotlin/com/apollostack/compiler/OperationTypeSpecBuilder.kt

```diff
@@ -1,26 +1,43 @@
 package com.apollostack.compiler
 
+import com.apollostack.compiler.ir.CodeGenerator
 import com.apollostack.compiler.ir.Field
-import com.cesarferreira.pluralize.singularize
+import com.apollostack.compiler.ir.Fragment
 import com.squareup.javapoet.TypeSpec
 import javax.lang.model.element.Modifier
 
 class OperationTypeSpecBuilder(
     val operationName: String,
-    val fields: List<Field>) {
+    val fields: List<Field>,
+    val allFragments: List<Fragment>) : CodeGenerator {
```

Comment:
How about to wrap this to next line for more readability, so it will look like this:
```
data class Fragment(
    val fragmentName: String,
    val source: String,
    val typeCondition: String,
    val fields: List<Field>,
    val fragmentsSpread: List<String>,
    val inlineFragments: List<String>,
    val fragmentsReferenced: List<String>
) : CodeGenerator {
  /** Returns a Java method that returns the interface represented by this Fragment object. */
  private fun toMethodSpec() =
      MethodSpec.methodBuilder(fragmentName.decapitalize())
          .returns(ClassName.get("", fragmentName.capitalize()))
          .addModifiers(Modifier.PUBLIC, Modifier.ABSTRACT)
          .build()
....
```